e2e/tests/notebook/notebook-download.spec.ts

```diff
@@ -15,6 +15,7 @@ describe('Jupyter notebook download test', () => {
     const checkDownloadDisabledState = async (wantDisabled: boolean) => {
       expect(await waitForFn(async () => {
         const downloadBtn = await modal.getDownloadButton();
+        // tslint:disable-next-line:triple-equals
         return wantDisabled == await getPropValue<boolean>(downloadBtn, 'disabled');
```

Comment:
Address tslint error. `==` is intentional.
`ERROR: 18:29  triple-equals  == should be ===`

Reply:
Why is == intentional here? Is getPropValue not returning an actual boolean? If not, I'd expect you could do

`wantDisabled == Boolean(await getPropValue<boolean>(downloadBtn, 'disabled'))`

Reply:
nit: would prefer coercing to boolean over bypassing the linter here, e.g. as I outlined in my comment:

> If you want to switch this to ===, please also make it !!(await getPropValue<boolean>(downloadBtn, 'disabled')); Unless you're 100% confident that your helper method properly handles this case.

Also, we should put the lint check into CircleCI, otherwise you'll be the only person that notices..

Reply:
Also, I can send that change for the `!!`